File: pacman-reloaded/src/main/java/launcher/Launcher.java
package launcher;

import com.uqbar.vainilla.DesktopGameLauncher;
import pacman.PacmanGame;

public class Launcher {

	public static void main(String[] args) throws Exception {
		new DesktopGameLauncher(new PacmanGame("application.xml")).launch();
	}
}

File: pacman-reloaded/src/main/java/launcher/launcher.rb
require 'java'
require 'active_support/all'

require '../config/app_config'
require '../pacman/pacman_game_builder'
require '../pacman/pacman_game'
require '../scenes/scene_builder'

java_import com.uqbar.vainilla.DesktopGameLauncher

c = AppConfig.new 'config/application.xml'

#Take this to pacman game initialization
scene = SceneBuilder.new(c).with_background('img/map.png').with_walkable_matrix('img/map.png').
    with_pacman('img/map.png', Resource.getImage(c['pacmanImg'])).build
game = PacmanGameBuilder.new(c).with_current_scene(scene).build

DesktopGameLauncher.new(game).launch